app/src/main/res/values/strings.xml

```diff
@@ -25,7 +25,7 @@
 
     <!-- Status warnings -->
     <string name="warn_5X601">A bug in Android 6.0.1 on the Nexus 5X prevents it from reading the hist bytes.</string>
-    <string name="warn_AV">Android 12 is untested.</string>
+    <string name="warn_AV">Android 14 is untested.</string>
```

Comment:
If you tested with Android 14, shouldn't Android 15 be untested then?

Reply:
In addition to changing the string, the condition should also be changed [in StatusFragment](https://github.com/nfcgate/nfcgate/blob/845650c26274c265973c8a324f8152daf163a14d/app/src/main/java/de/tu_darmstadt/seemoo/nfcgate/gui/fragment/StatusFragment.java#L161).

Reply:
Not sure if me testing is it on android 14 is sufficient testing heh.

Reply:
If you report it working on Android 14 in tag mode, that is sufficient.

Reply:
Marked as 15 being untested.